pkg/services/store/service.go

```diff
@@ -53,9 +68,15 @@ func ProvideService(sql *sqlstore.SQLStore, features featuremgmt.FeatureToggles,
 	_ = os.MkdirAll(storage, 0700)
 
 	if features.IsEnabled(featuremgmt.FlagStorageLocalUpload) {
+		upload := filepath.Join(storage, "upload")
+		grafanaStorageLogger.Info("inside provide service", "flag", true)
```

Comment:
For backend do we typically want to keep these types of logs for merged in code?

Reply:
oof no will clean up